frontends/open-discussions/package.json

```diff
@@ -184,8 +183,8 @@
     "codecov": "CODECOV=1 ./scripts/js_test.sh",
     "watch": "WATCH=1 ./scripts/js_test.sh",
     "typecheck": "flow check ./src",
-    "fmt-fix": "LOG_LEVEL= prettier-eslint --write --no-semi --ignore $PROJECT_CWD/'frontends/open-discussions/src/flow/**/*.js' $PROJECT_CWD/'frontends/open-discussions/src/**/*.js' $PROJECT_CWD/'frontends/open-discussions/scss/**/*.scss'",
-    "fmt-check": "prettier-eslint --list-different --no-semi --ignore $PROJECT_CWD/'frontends/open-discussions/src/flow/**/*.js' $PROJECT_CWD/'frontends/open-discussions/src/**/*.js' $PROJECT_CWD/'frontends/open-discussions/scss/**/*.scss'",
+    "fmt-fix": "LOG_LEVEL= prettier-eslint --eslint-path $INIT_CWD/node_modules/eslint --write --no-semi --ignore $PROJECT_CWD/'frontends/open-discussions/src/flow/**/*.js' $PROJECT_CWD/'frontends/open-discussions/src/**/*.js' $PROJECT_CWD/'frontends/open-discussions/scss/**/*.scss'",
+    "fmt-check": "prettier-eslint --eslint-path $INIT_CWD/node_modules/eslint --list-different --no-semi --ignore $PROJECT_CWD/'frontends/open-discussions/src/flow/**/*.js' $PROJECT_CWD/'frontends/open-discussions/src/**/*.js' $PROJECT_CWD/'frontends/open-discussions/scss/**/*.scss'",
```

Comment:
Now that the root directory has `prettier-eslint`, too, open-discussions it seems to be having trouble finding the correct eslint file for open-discussions, so I added the `--eslint-path` bit here.

(_open-discussions is on a much older version of `eslint-config-mitodl`_)